Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h

```diff
@@ -586,10 +586,12 @@ auto InvMassXicToPiKP(const T& candidate)
 {
   return InvMassLcpiKp(candidate);
 }
+
 } // namespace hf_cand_prong3
 
 // 3-prong decay candidate table
 DECLARE_SOA_TABLE(HfCandProng3Base, "AOD", "HFCANDP3BASE", //!
+                  o2::soa::Index<>,
```

Comment:
What does this do and why is it needed?

Reply:
from what I understood this is what is needed in order to match the candidate index with the track index. @rspijkers do you recall how this came up?

Reply:
hi @aalkin can you help us with this?

Reply:
@vkucera @ginnocen this is what provides `.globalIndex()` call for the table. This is a simple enumerator, it is only needed to get a global index for a row, which, in turn is needed if one wants to build a combined index table. Regardless, there is no cost to have this column in table.

Reply:
thanks @aalkin for the clarification.